03LessonThree/homework/git-scm.html

```diff
@@ -3,58 +3,68 @@
 
 <head>
     <title>Git</title>
-    <link href="css/git-style.css" rel="stylesheet"/>
+    <link href="css/git-style.css" rel="stylesheet" />
 </head>
 
 <body>
-    <div id="header">
-        <div id="nav">
+    <header>
+        <nav>
             <span><a href="#">Home</a></span>
             <span><a href="#">How does it work</a></span>
             <span><a href="#">Contact Us</a></span>
-        </div>
-    </div>
-    <div id="main">
-        <div id="section">
-            <img src="http://git-scm.com//images/logo@2x.png" alt="git logo"/>
+        </nav>
+    </header>
+    <main>
+        <section>
+            <img src="http://git-scm.com//images/logo@2x.png" alt="git logo" />
             <h2>Welcome to Git</h2>
             <p>Git is a free and open source distributed version control system designed to handle everything from small to very large projects with speed and efficiency. </p>
-        </div>
-        <div id="section">
-            <div id="article">
+        </section>
+        <section>
+            <article>
                 <h3>About</h3>
-                <p>The Git feature that really makes it stand apart from nearly every other SCM out there is its branching model. Git allows and encourages you to have multiple local branches that can be entirely independent of each other. The creation, merging, and deletion of those lines of development takes seconds. This means that you can do things like:</p>
+                <p>The Git feature that really makes it stand apart from nearly every other SCM out there is its branching model. Git allows and encourages you to have multiple local branches that can be entirely independent of each other. The creation,
+                    merging, and deletion of those lines of development takes seconds. This means that you can do things like:</p>
                 <ul>
                     <li>Frictionless Context Switching. Create a branch to try out an idea, commit a few times, switch back to where you branched from, apply a patch, switch back to where you are experimenting, and merge it in.</li>
                     <li>Role-Based Codelines. Have a branch that always contains only what goes to production, another that you merge work into for testing, and several smaller ones for day to day work.</li>

```

Comment:
Fantastic! Thank you for formatting the HTML document, indenting your html markup like you did here is very useful to you and other developers looking at the document in the future.